tests/e2e/lib/reporters/slack.js

```diff
@@ -5,14 +5,14 @@ import { readFileSync, createReadStream } from 'fs';
 import { WebClient, ErrorCode } from '@slack/web-api';
 import config from 'config';
 
-const { GITHUB_EVENT_PATH, GITHUB_RUN_ID, LATEST_GUTENBERG } = process.env;
+const { GITHUB_EVENT_PATH, GITHUB_RUN_ID, GUTENBERG } = process.env;
 
 export default class SlackReporter {
 	constructor() {
 		const token = config.get( 'slackToken' );
 		this.webCli = new WebClient( token );
 		this.runURL = `https://github.com/Automattic/jetpack/actions/runs/${ GITHUB_RUN_ID }`;
-		this.runType = LATEST_GUTENBERG ? 'with latest :gutenberg:' : 'All';
+		this.runType = GUTENBERG === 'latest' ? 'with latest :gutenberg:' : 'All';
```

Comment:
Should this match the changes to the job name in the yaml file?

Reply:
Uh, match how? Sorry, not seeing/grokking it :sweat_smile:

Reply:
So before we had jobs named "with latest Gutenberg" and "All". This is outputting more or less same thing.

Now our jobs are named "E2E tests: with latest plugin" and "E2E tests: with no plugin". It might be confusing if the slack message still says "All" when it's referring to the "with no plugin" test.

Reply:
Ah, gotcha! Yep, I'll change that :+1: